Started by user [8mha:////4Gm8z8svCaXKqFzr1Z5zntsQFmxEgC1XfjqScehTkFs8AAAAlx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzWEgZu/dLi1CL9xJTczDwAj6GcLcAAAAA=[0mnazar
Running as SYSTEM
Building remotely on [8mha:////4Gb5jqnYN5atl4H6TvVp5YbnrSRqULYVnZcIQhut4W5hAAAAnx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzOEgZh/eT83ILSktQi/TBfI93inMSyVAA+LWVRyAAAAA==[0mVM2-slave (linux_vm2_slave) in workspace /home/user1/workspace/ansible-docker
[ansible-docker] $ /bin/sh -xe /tmp/jenkins2269727568571851263.sh
+ build_number=63
+ sudo touch /etc/ansible/playbook-docker.yml
+ sudo chmod 755 /etc/ansible/playbook-docker.yml
+ sudo tee /etc/ansible/playbook-docker.yml
+ sudo sed -i s/buildnumber/63/g /etc/ansible/playbook-docker.yml
[ansible-docker] $ sshpass ******** ansible-playbook /etc/ansible/playbook-docker.yml -f 5 -u user1 -k

PLAY [ansibleslave] ************************************************************

TASK [Gathering Facts] *********************************************************
[WARNING]: Platform linux on host 172.31.32.48 is using the discovered Python
interpreter at /usr/bin/python, but future installation of another Python
interpreter could change this. See https://docs.ansible.com/ansible/2.9/referen
ce_appendices/interpreter_discovery.html for more information.
ok: [172.31.32.48]

TASK [Creates directory] *******************************************************
changed: [172.31.32.48]

TASK [setting up gitrepo] ******************************************************
changed: [172.31.32.48]

TASK [Get the project from github] *********************************************
changed: [172.31.32.48]

PLAY RECAP *********************************************************************
172.31.32.48               : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

[ansible-docker] $ /bin/sh -xe /tmp/jenkins7350848685184305169.sh
+ build_number=63
+ sudo touch /etc/ansible/playbook-docker-deploy.yml
+ sudo chmod 755 /etc/ansible/playbook-docker-deploy.yml
+ sudo tee /etc/ansible/playbook-docker-deploy.yml
+ sudo sed -i s/buildnumber/63/g /etc/ansible/playbook-docker-deploy.yml
[ansible-docker] $ sshpass ******** ansible-playbook /etc/ansible/playbook-docker-deploy.yml -f 5 -u user1 -k

PLAY [ansibleslave] ************************************************************

TASK [Gathering Facts] *********************************************************
[WARNING]: Platform linux on host 172.31.32.48 is using the discovered Python
interpreter at /usr/bin/python, but future installation of another Python
interpreter could change this. See https://docs.ansible.com/ansible/2.9/referen
ce_appendices/interpreter_discovery.html for more information.
ok: [172.31.32.48]

TASK [create dockerfile] *******************************************************
changed: [172.31.32.48]

TASK [add content to dockerfile] ***********************************************
ok: [172.31.32.48]

TASK [build container] *********************************************************
changed: [172.31.32.48]

TASK [run the container] *******************************************************
changed: [172.31.32.48]

TASK [start the container] *****************************************************
changed: [172.31.32.48]

TASK [run the container] *******************************************************
changed: [172.31.32.48]

PLAY RECAP *********************************************************************
172.31.32.48               : ok=7    changed=5    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

[ansible-docker] $ /bin/sh -xe /tmp/jenkins3397672276956121877.sh
+ build_number=63
+ sudo touch /etc/ansible/playbook-docker-project.yml
+ sudo chmod 755 /etc/ansible/playbook-docker-project.yml
+ sudo tee /etc/ansible/playbook-docker-project.yml
+ sudo sed -i s/buildnumber/63/g /etc/ansible/playbook-docker-project.yml
[ansible-docker] $ ansible-playbook /etc/ansible/playbook-docker-project.yml -f 5
ERROR! We were unable to read either as JSON nor YAML, these are the errors we got from each:
JSON: No JSON object could be decoded

Syntax Error while loading YAML.
  could not find expected ':'

The error appears to be in '/etc/ansible/playbook-docker-project.yml': line 9, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

      container_id=$(docker ps -a|grep project-63|awk {'print $1'})
      project_name=$(ls /home/user1/project_63/)
      ^ here
FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:262)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:232)
	at jenkins.tasks.SimpleBuildStep.perform(SimpleBuildStep.java:123)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:80)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:804)
	at hudson.model.Build$BuildExecution.build(Build.java:197)
	at hudson.model.Build$BuildExecution.doRun(Build.java:163)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:514)
	at hudson.model.Run.execute(Run.java:1907)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
ERROR: Ansible playbook execution failed
Finished: FAILURE
